src/components/store/OrderSummary/index.tsx

```diff
@@ -57,6 +49,12 @@ const OrderItemPreview = ({ item, orderStatus }: OrderItemPreviewProps) => {
           alt="Store item picture"
           fill
         />
+        {showFulfilled && item.fulfilled ? (
+          <div className={styles.badge}>{showNotFulfilled ? 'Picked up' : 'Already picked up'}</div>
+        ) : null}
+        {showNotFulfilled && !item.fulfilled ? (
+          <div className={`${styles.badge} ${styles.notAvailable}`}>Not available</div>
```

Comment:
Not available makes me think that it's not in stock, can we do something like not picked up maybe? Could also ask others on what wording is best

Reply:
yeah it should only show for partially fulfilled events, so it wasn't in stock

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/e001e245-16f7-420b-ace4-b9add606165e)